### PR TITLE
Fix a couple of typos in some of the pages

### PR DESCRIPTION
**Pull request by vandycknick**

### CODE REVIEW

File: src/pages/useDebounce.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ sandbox: https://codesandbox.io/s/711r1zmq50
...
+This hook allows you to debounce any fast changing value. The debounced value will only reflect the latest value when the useDebounce hook has not been called for the specified time period. When used in conjunction with useEffect, as we do in the recipe below, you can easily ensure that expensive operations like API calls are not executed too frequently. The example below allows you to search the Marvel Comic API and uses useDebounce to prevent API calls from being fired on every keystroke. Be sure to check out the [CodeSandbox demo](https://codesandbox.io/s/711r1zmq50) for this one. Hook code and inspiration from [github.com/xnimorz/use-debounce](https://github.com/xnimorz/use-debounce).
```
Comment (vandycknick) [inline src/pages/useDebounce.md:10]: conjuction -> conjunction

File: src/pages/useMemo.md [hunk 2]
```diff
@@ -27,4 +27,4 @@ code:
...
+React has a built-in hook called useMemo that allows you to memoize expensive functions so that you can avoid calling them on every render. You simple pass in a function and an array of inputs and useMemo will only recompute the memoized value when one of the inputs has changed. In our example below we have an expensive function called computeLetterCount (for demo purposes we make it slow by including a large and completely unnecessary loop). When the current selected word changes you'll notice a delay as it has to recall computeLetterCount on the new word. We also have a separate counter that gets incremented every time the increment button is clicked. When that counter is incremented you'll notice that there is zero lag between renders. This is because computeLetterCount is not called again. The input word hasn't changed and thus the cached value is returned. You'll probably want to check out the [CodeSandbox demo](https://codesandbox.io/s/jjxypyk86w) so you can see for yourself.
```
Comment (vandycknick) [inline src/pages/useMemo.md:30]: everytime -> every time

File: src/pages/useScript.md [hunk 3]
```diff
@@ -41,4 +41,4 @@ code:
...
+This hook makes it super easy to dynamically load an external script and know when its loaded. This is useful when you need to interact with a 3rd party library (Stripe, Google Analytics, etc) and you'd prefer to load the script when needed rather then include it in the document head for every page request. In the example below we wait until the script has loaded successfully before calling a function declared in the script. If you're interested in seeing how this would look if implemented as a Higher Order Component then check out the [source of react-script-loader-hoc](https://github.com/sesilio/react-script-loader-hoc/blob/master/src/index.js). I personally find it much more readable as a hook. Another advantage is because it's so easy to call the same hook multiple times to load multiple different scripts, unlike the HOC implementation, we can skip adding support for passing in multiple src strings.
```
Comment (vandycknick) [inline src/pages/useScript.md:44]: libary -> library

File: src/pages/useTheme.md [hunk 4]
```diff
@@ -11,4 +11,4 @@ links:
...
+This hook makes it easy to dynamically change the appearance of your app using CSS variables. You simply pass in an object containing key/value pairs of the CSS variables you'd like to update and the hook updates each variable in the document's root element. This is useful in situations where you can't define styles inline (no pseudo class support) and there are too many style permutations to include each theme in your stylesheet (such as a web app that lets users customize the look of their profile). It's worth noting that many css-in-js libraries support dynamic styles out of the box, but it's interesting to experiment with how this can be done with just CSS variables and a React Hook. The example below is intentionally very simple, but you could imagine the theme object being stored in state or fetched from an API. Be sure to check out the [CodeSandbox demo](https://codesandbox.io/s/15mko9187) for a more interesting example and to see the accompanying stylesheet.
```
Comment (vandycknick) [inline src/pages/useTheme.md:14]: psudeoclass -> pseudo class

File: src/pages/useWhyDidYouUpdate.md [hunk 5]
```diff
@@ -7,6 +7,6 @@ sandbox: https://codesandbox.io/s/kx83n7201o
...
 ---
 
-This hook makes it easy to see which prop changes are causing a component to re-render. If a function is particularly expensive to run and you know it renders the same results given the same props you can use the `React.memo` higher order component, as we've done with the `Counter` component in the below example. In this case if you're still seeing re-renders that seem uneccessary you can drop in the `useWhyDidYouUpdate` hook and check your console to see which props changed between renders and view their previous/current values. Pretty nifty huh?
+This hook makes it easy to see which prop changes are causing a component to re-render. If a function is particularly expensive to run and you know it renders the same results given the same props you can use the `React.memo` higher order component, as we've done with the `Counter` component in the below example. In this case if you're still seeing re-renders that seem unnecessary you can drop in the `useWhyDidYouUpdate` hook and check your console to see which props changed between renders and view their previous/current values. Pretty nifty huh?
```
Comment (vandycknick) [inline src/pages/useWhyDidYouUpdate.md:10]: uneccessary -> unnecessary

**Comment by gragland** [comment]
Thanks @nickvdyck! Will get this merged soon.

**Pull request merged**